src/Compilers/CSharp/Portable/BoundTree/DecisionTree.cs

```diff
@@ -197,6 +198,11 @@ internal DecisionTree ComputeDecisionTree()
                         else
                         {
                             AddToDecisionTree(result, label);
+                            if (label.Guard == null || label.Guard.ConstantValue == ConstantValue.True)
+                            {
+                                // Only unconditional switch labels contribute to subsumption
+                                AddToDecisionTree(subsumptionTree, label);
```

Comment:
I'm a bit confused -- does `CheckSubsumed` itself modify the tree? Otherwise, haven't we just added the same tree as on line 200?